packages/edit-site/src/components/global-styles/screen-colors.js

```diff
@@ -263,40 +52,12 @@ function ScreenColors( { name, variation = '' } ) {
 				<VStack spacing={ 10 }>
 					<Palette name={ name } />
```

Comment:
I'm not sure if it's out of the scope of this PR, but I found [this issue](https://github.com/WordPress/gutenberg/issues/36541#issuecomment-1480882023). Is it easier with this refactor to know if we are on the global vs the block level to hide the palette component if I'm on the block level?

Reply:
Yeah, it's a bit independent. The current PR doesn't touch the "palette" global styles screens. It only refactors the UI elements used to actually set the styles.